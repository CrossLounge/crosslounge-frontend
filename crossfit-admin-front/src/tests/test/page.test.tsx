import Test from '@/app/test/page';
import { render, screen } from '@testing-library/react';

describe('<Test />', () => {
    it('renders a heading', () => {
      const { container } = render(<Test />);
  
      const home = screen.getByText('테스트 아이템 입니다.');
  
      expect(home).toBeInTheDocument();
      expect(container).toMatchSnapshot();
    });
  });