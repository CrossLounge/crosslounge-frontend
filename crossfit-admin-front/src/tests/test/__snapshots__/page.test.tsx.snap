// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test /> renders a heading 1`] = `
<div>
  <div>
    테스트 아이템 입니다.
  </div>
</div>
`;
